compiler/optimizer/Inliner.cpp

```diff
@@ -5011,6 +5011,18 @@ bool TR_InlinerBase::inlineCallTarget2(TR_CallStack * callStack, TR_CallTarget *
             _disableTailRecursion = true;
          }
       }
+   else if (comp()->getOption(TR_FullSpeedDebug) && tif->crossedBasicBlock())
```

Comment:
Yes I am fine with @ncough doing it as part of his PR. Otherwise this change is good and I will merge it now.